### PR TITLE
so... this is hunger?

### PR DESCRIPTION
**Pull request by dannenberg**

### CODE REVIEW

**Comment by dannenberg** [comment]
@GTDev87 peep that interface i gave you for the hud (getCurrentHungerLevel and getMaxHungerLevel)

**Comment by dannenberg** [comment]
also is this how our hunger should?
every N secs drop half a sandwich

**Review by orez- — CHANGES_REQUESTED** [review]
eeeesh

**Comment by GTDev87** [comment]
you can merge it.  only things i want is flipping "hunger level" and fullness where `(fullness === 0) === death` also the timer i feel coupled with the player may be a bit dangerous.

File: src/js/player.js [hunk 1]
```diff
@@ -1,10 +1,28 @@
 const Actor = require('./actor');
 
 const MOVE_SPEED = 150;
+
+const HUNGER_INCREASE_RATE = 200; // millis
+// How many hunger before death.
+const STARVING_TO_DEATH = 8;
```
Comment (dy-dx) [inline src/js/player.js:7]: let's just make this 100

Comment (dannenberg) [reply src/js/player.js:7]: see https://github.com/Hamfisted/ld38/pull/21

**Pull request closed without merging**